test/membrane_live_web/controllers/webinar_controller_test.exs

```diff
@@ -44,21 +40,19 @@ defmodule MembraneLiveWeb.WebinarControllerTest do
     test "renders webinar when data is valid", %{conn: conn} do
       conn = post(conn, Routes.webinar_path(conn, :create), webinar: @create_attrs)
 
-      assert %{"viewer_link" => viewer_link, "moderator_link" => moderator_link} =
+      assert %{"viewer_link" => viewer_link, "moderator_link" => _moderator_link} =
                json_response(conn, 201)["webinar_links"]
 
-      webinar = Webinars.get_webinar_by_link(viewer_link)
-      id = webinar.id
+      uuid = get_uuid_from_link(viewer_link)
+      webinar = Webinars.get_webinar(uuid)
```

Comment:
We could add an assertion about whether `moderator_link` has a correct suffix and also whether both links have correct prefixes.

Reply:
Done 🏋️‍♂️